Library/Homebrew/extend/os/linux/requirements/x11_requirement.rb

```diff
@@ -0,0 +1,3 @@
+class X11Requirement
+  default_formula "linuxbrew/xorg/xorg"
```

Comment:
I'd rather avoid any `linuxbrew` tap references until we are further along with the migration to merge Linuxbrew/brew and Homebrew/brew.

Reply:
I've removed `extend/os/linux/requirements/x11_requirement.rb`